packages/core/rollup.config.js

```diff
@@ -2,13 +2,14 @@ import commonjs from "@rollup/plugin-commonjs";
 import json from "@rollup/plugin-json";
 import { nodeResolve } from "@rollup/plugin-node-resolve";
 
+import { extractExports } from "../../extractExports.js";
+
+import * as packageJson from "./package.json" assert { type: "json" };
```

Comment:
in file without `imports` it will be just `require` 😄

Reply:
Yes, I could also convert all rollup configs to `cjs`...

Reply:
rollup config examples all use ESM so I thought I'd keep it like that.